README.md

```diff
@@ -24,7 +24,9 @@ source code repository.
 Run master and workers against your local Olympia docker container (by IP
 address, which may be different for you).
 
-    SITE_UNDER_TEST=http://192.168.59.103 docker-compose up -d
+    SITE_UNDER_TEST=http://$(docker-machine ip default) \
+        MASTER_HOST=$(docker-machine ip default) \
+        docker-compose up -d
 
 Open the Locust dashboard at http://192.168.59.103:8089/
```

Comment:
you would need to update this as well. Maybe `open http://$(docker-machine ip default)` ? The `open` command is maybe mac/linux only